pages/client-libraries/python.mdx

```diff
@@ -645,65 +645,128 @@ The `Session.run()` method is most commonly used for `LOAD CSV` clause to preven
+If you are running transactions in parallel, you should avoid [implicit transactions](#implicit-transactions) because you can't control the execution process, and there are no retries.
+
+You can use the [managed transactions](#managed-transactions) or [explicit transactions](#explicit-transactions) to handle the conflicting transactions. Explicit API provides full control of the process, and it is recommended for production use and handling conflicts. 
+
+Here is an example of how to handle conflicting transactions in explicit API:
+
+```Python
+def process_chunk(query, create_list, max_retries=100, initial_wait_time=0.200, backoff_factor=1.1, jitter=0.1):
+    session = GraphDatabase.driver(HOST_PORT, auth=("", "")).session()
+    for attempt in range(max_retries):
+        try:
+            with session.begin_transaction() as tx:
+                tx.run(query, {"batch": create_list})
+                tx.commit()
+                break
+        except TransientError as te:
+            jitter = random.uniform(0, jitter) * initial_wait_time 
+            wait_time = initial_wait_time * (backoff_factor ** attempt) + jitter
+            print(f"Commit failed on attempt {attempt+1}. Retrying in {wait_time} seconds...")
+            time.sleep(wait_time)
+        except Exception as e:
+            print(f"Failed to execute transaction: {e}")
+            session.close()
+            raise e
 ```
 
-The following example shows how to use the client with transaction functions:
+In the example above, we are using the `begin_transaction()` method to start a transaction, and then we are running the query inside the transaction. If the transaction fails with a `Transient error,` the transaction will be retried using the retry strategy. Otherwise, another error occurred, and the transaction should be aborted. 
 
-```python
-import asyncio
-from neo4j import AsyncGraphDatabase
+The essential aspects of the retry strategy are the following arguments: 
 
+- `max_retries` - the maximum number of retries before the transaction will be aborted with a timeout error. This number should be set based on the expected number of conflicts and the time that the transaction will be running.
+if the commit takes a second and you have 120 retries, that means that the transaction will be running for 2 minutes + waiting time.
+- `initial_wait_time` - the time that the transaction will wait before the first retry. The first time a transaction fails with conflict, it will wait for `initial_wait_time` seconds before the first retry.
+- `backoff_factor` - the factor by which the retry delay will be multiplied after each retry. This number should not be too high because it can lead to long-running transactions; if commits are small and fast, avoid exponential backoff.
+- `jitter` - the factor by which the retry delay will be randomized after each retry. If there are a lot of transactions running in parallel, it is recommended to use jitter to avoid the thundering herd problem.
```

Comment:
```suggestion
- `jitter` - the factor by which the retry delay will be randomized after each retry. If there are a lot of transactions running in parallel, it is recommended to use `jitter` to avoid the thundering herd problem.
```